Code/Tools/ProjectManager/Source/GemRepo/GemRepoItemDelegate.cpp

```diff
@@ -102,14 +112,56 @@ namespace O3DE::ProjectManager
 
         painter->drawText(repoCreatorRect, Qt::TextSingleLine, repoCreator);
 
-        // Repo update
+        // Badge
         currentHorizontalOffset += sectionSize;
-        sectionSize = m_headerWidget->m_header->sectionSize(static_cast<int>(HeaderOrder::Update));
+        sectionSize = m_headerWidget->m_header->sectionSize(static_cast<int>(HeaderOrder::Badge));
+        auto badgeType = GemRepoModel::GetBadgeType(modelIndex);
+        const QPixmap* badge = nullptr;
+        QString badgeText;
+        if (badgeType == GemRepoInfo::BadgeType::BlueBadge)
+        {
+            badge = &m_blueBadge;
+            badgeText = tr("O3DE Official");
+        }
+        else if (badgeType == GemRepoInfo::BadgeType::GreenBadge)
+        {
+            badge = &m_greenBadge;
+
+            // this text should be made dynamic at some point
+            badgeText = tr("O3DF Recommended");
+        }
+
+        if (badge)
+        {
+            const QRect badgeRect = CalcBadgeRect(contentRect);
+            painter->drawPixmap(badgeRect, m_blueBadge);
+
+            painter->setFont(standardBoldFont);
+
+            QRect badgeLabelRect = GetTextRect(standardBoldFont, badgeText, s_fontSize);
+            badgeLabelRect.moveTo(currentHorizontalOffset + s_badgeLeftMargin,
+                contentRect.center().y() - (badgeLabelRect.height() / 2) - 1);
+            badgeLabelRect = painter->boundingRect(badgeLabelRect, Qt::TextSingleLine, badgeText);
+            painter->drawText(badgeLabelRect, Qt::TextSingleLine, badgeText);
+
+            painter->setFont(standardFont);
+        }
+
+        // Last updated
+        currentHorizontalOffset += sectionSize;
+        sectionSize = m_headerWidget->m_header->sectionSize(static_cast<int>(HeaderOrder::Updated));
+        auto lastUpdated = GemRepoModel::GetLastUpdated(modelIndex);
+
+        // get the month day and year in the preferred locale's format (QLocale defaults to the OS locale)
+        QString monthDayYear = lastUpdated.toString(QLocale().dateFormat(QLocale::ShortFormat));
 
-        QString repoUpdatedDate = GemRepoModel::GetLastUpdated(modelIndex).toString(RepoTimeFormat);
+        // always show 12 hour + minutes + am/pm
+        QString hourMinuteAMPM = lastUpdated.toString("h:mmap");
```

Comment:
Time zones are important here.
What if the user experiences a change in time on their machine due to daylight saving time and the value is off by an hour.
Or the user travels to another time zone.

Reply:
That's a good call, though because this is an existing field that didn't use UTC time it might require some extra logic to still support old fields in the old format.